integration-tests/tests/list_test.go

```diff
@@ -36,8 +36,8 @@ type listSuite struct {
 	common.SnappySuite
 }
 
-// FIXME: hardcoded for now
-var verRegexp = "16.04.*"
+// FIXME: hardcoded or sideload
```

Comment:
hey @stevenwilkin, the version was still hardcoded, probably the comment wasn't clear enough. I've taken the opportunity to fix the regexp so that it will accept future versions, please take another look.

Thanks! :blush: